src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -2683,7 +2683,7 @@ private function registerHtmlSanitizerConfiguration(array $config, ContainerBuil
             }
 
             if ($sanitizerConfig['allow_all_static_elements']) {
```

Comment:
what about renaming this to allow_all_static_elements then? we can still do it imho